### PR TITLE
(do not merge) prosthetic limb rework???

### PR DESCRIPTION
**Pull request by xuv8r**
## About The Pull Request

this is supposed to make prosthetic limbs better. it only kind of works
image
why is it broken. why is left limb at the bottom. what do you MEAN, tgui.

anyway basically instead of it being random-limb-or-all-limbs, you can choose per-limb what limb type you want to be robotic

this could stand to get cleaned up way way more, but it's kind of a proof-of-concept

todo (probably):
-disentangle choosable limbs from the prosthetics quirk
-put the limb choice dropdowns on a new screen
-instead of reading "digitigrade" as a separate type, maybe just read whether the player already has digitigrade preference selected
-fuck surplus limbs why would you ever take them over the standard
-rename "normal" option to something else?
-write a real changelog when the PR is for real ready

basically only posting this as a PR so other people can look at it 

## Why It's Good For The Game

prosthetic limb being random is just silly. what if you want to rp as someone who got a specific limb hacked off. or whatever. or maybe you just want some cool robot limbs.

## Changelog

:cl:
add: "prosthetics" quirk that fucks up your whole life and calls you a fool for trying to code it
/:cl:

### CODE REVIEW

**Comment by Or-Fi-S** [comment]
*One would take surplus limbs for the same reason one would still be taking negative traits: it says something important to them that they have this bad situation going on

**Comment by xuv8r** [comment]
hmm. i suppose hypothetically this could also include "don't have a limb there at all". not sure if i want to incorporate that

**Comment by xuv8r** [comment]
additional bug discovered: digitigrade robotic legs do not prevent shoes from being equipped at spawn
